### PR TITLE
add nvtx for host code.

### PR DESCRIPTION
**Pull request by csukuangfj**
ready for review.

### CODE REVIEW

File: k2/csrc/host/rmepsilon.cc [hunk 1]
```diff
@@ -22,6 +22,8 @@
 #include "k2/csrc/host/fsa.h"
 #include "k2/csrc/host/properties.h"
 #include "k2/csrc/host/util.h"
+#include "k2/csrc/macros.h"
```
Comment (qindazhu) [inline k2/csrc/host/rmepsilon.cc:25]: @csukuangfj Can you revert changes in `rmepsilon.cc`, I can add NVTX stuff in my next PR (I have a big change in this file).

**Comment by csukuangfj** [comment]
Will revert it




Thursday, 3 December 2020, 11:24 +0800 from notifications@github.com  <notifications@github.com>:

File: k2/csrc/host/rmepsilon.cc (hunk 1 above)
Comment (csukuangfj) [reply k2/csrc/host/rmepsilon.cc]: Fixed.

Comment (qindazhu) [reply k2/csrc/host/rmepsilon.cc]: Many thanks

**Comment by qindazhu** [comment]
Merging

**Pull request merged**